include/graphtyper/typer/sample_call.hpp

```diff
@@ -45,7 +45,7 @@ class SampleCall
   std::pair<uint16_t, uint16_t> get_gt_call() const;
   uint8_t get_gq() const;
   uint8_t get_lowest_phred_not_with(uint16_t allele) const;
-  int8_t check_filter(long gq) const;
+  long check_filter(long gq) const;
```

Comment:
Why this change?

I would actually propose replacing all instances of `long` and `int` with fixed-size counter-parts or `size_t` when something is counted. This is actually quite important for serialised data structures as `long` and `int` have different sizes on different platforms.

Reply:
Previous code had a "bugprone-signed-char-misuse" warning from tidy. I don't actually see anything that could potentially be a bug here but it's just some `int8_t` is `signed char` fun.

I personally think `int`/`long` are fine defaults when exact size doesn't matter. But sure we can use fixed-size types in serialized data structures though.